src/modules/zenoh/zenoh_config.cpp

```diff
@@ -217,8 +343,17 @@ void Zenoh_Config::getNetworkConfig(char *mode, char *locator)
 	// If file opened successfully, then read the file
 	if (fp) {
 		fgets(buffer, NET_CONFIG_LINE_SIZE, fp);
-		const char *config_locator = get_csv_field(buffer, 2);
-		char *config_mode = (char *)get_csv_field(buffer, 1);
+		const char *fields[2];
```

Comment:
`client;` is/was a valid config for the networking, but with this change that would break now.
We can also update the default config though.